### PR TITLE
feat: [IDP-2089] :  Added the information on troubleshooting and use of k8s plugin

### PR DESCRIPTION
**Pull request by Debanitrkl**
# Harness Developer Pull Request
Added the information on troubleshooting and use of k8s plugin

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/internal-developer-portal/plugins/available-plugins/kubernetes.md [hunk 1]
```diff
@@ -150,6 +150,105 @@ You can write your own custom label selector query that Backstage will use to lo
...
 'backstage.io/kubernetes-label-selector': 'app=my-app,component=front-end'
 ```
+## Troubleshooting
+
+### Kubernetes is not showing up on Service Entities
+
+This can be debugged by checking whether your Kubernetes cluster are connected to Backstage as follows:
+
+```sh
+curl --location --request POST '{{backstage-backend-url}}:{{backstage-backend-port}}/api/kubernetes/services/:service-entity-name' \
```
Comment (sandeepa-kv-jois) [inline docs/internal-developer-portal/plugins/available-plugins/kubernetes.md:160]: How would they know the values for {{backstage-backend-url}}:{{backstage-backend-port}} ?

**Review by sandeepa-kv-jois — COMMENTED** [review]
Looks good except debugging review comment provided

File: docs/internal-developer-portal/plugins/available-plugins/kubernetes.md (hunk 1 above)
Comment (Debanitrkl) [reply docs/internal-developer-portal/plugins/available-plugins/kubernetes.md:160]: Aah okay I think this isn't possible with IDP rather the connector should have access will mention that

Comment (Debanitrkl) [reply docs/internal-developer-portal/plugins/available-plugins/kubernetes.md]: Updated it

**Review by devesh-harness — APPROVED** [review]
LGTM

**Pull request merged**